cli/admin/call.go

```diff
@@ -0,0 +1,193 @@
+package admin
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/project"
+
+	"github.com/spf13/pflag"
+
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/templates"
+)
+
+const (
+	argTypeString = "string"
+	argTypeNumber = "number"
+	argTypeBool   = "boolean"
+)
+
+type FuncCallArg struct {
```

Comment:
https://github.com/tarantool/cartridge-cli/pull/378/files#r505575195

>  comment every struct field

Everything  is clear with `argTypeString ` - it's self-obvious, but struct should be commented